pkg/types/validation/installconfig.go

```diff
@@ -236,6 +238,11 @@ func validatePlatform(platform *types.Platform, fldPath *field.Path, openStackVa
 	if platform.VSphere != nil {
 		validate(vsphere.Name, platform.VSphere, func(f *field.Path) field.ErrorList { return vspherevalidation.ValidatePlatform(platform.VSphere, f) })
 	}
+	if platform.BareMetal != nil {
+		validate(baremetal.Name, platform.BareMetal, func(f *field.Path) field.ErrorList {
```

Comment:
can we compact this to one line like the others.

Reply:
gofmt fails if this is on one line, perhaps because it's longer than the others?